src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -4950,6 +4970,13 @@ void calcMeanAirTemps(EnergyPlusData &state,
             }
         }
     }
+    auto const &thisZone = state.dataHeatBal->Zone(zoneNum);
+    if (thisZone.ReportWBGT) {
```

Comment:
If you move `ReportWBGT` into `thisAirRpt` (`state.dataHeatBal->ZnAirRpt` and `spaceAirRpt`) then it will work for both here.

Reply:
good idea. I'll move it to be one of the fields in `thisAirRpt`